ci/tox/docker-compose.yml

```diff
@@ -2,18 +2,27 @@ version: '3'
 services:
   tox:
     image: gitmachete/ci-tox:git${GIT_VERSION}-python${PYTHON_VERSION}-${DIRECTORY_HASH:-unspecified}
+#    user: "${USER_ID?}:${GROUP_ID?}"
```

Comment:
Hmmm quite a lot of code is commented out on this PR, and apparently no new code added... does this PR contain an actual PoC to be considered? 🤔

Reply:
oh! so no working version has ever been achieved on this PR? if so, then discard the PR to clean the stage; otherwise, restore whatever was working